pkg/master/master_test.go

```diff
@@ -526,7 +526,7 @@ func testInstallThirdPartyAPIListVersion(t *testing.T, version string) {
 			setupEtcdList(fakeClient, "/ThirdPartyResourceData/company.com/foos/default", test.items)
 		}
 
-		resp, err := http.Get(server.URL + "/apis/company.com/" + version + "/namespaces/default/foos")
+		resp, err := http.Get(server.URL + "/apis/company.com/group/" + version + "/namespaces/default/foos")
```

Comment:
@bgrant0607  I *did* make a proposal, please see https://github.com/kubernetes/kubernetes/pull/14522 and argue there